content/people/weyers-sandra.yml

```diff
@@ -1,2 +1,2 @@
 firstName: Sandra
-lastName: Weyers
+lastName: Weyand
```

Comment:
when changing a name *after* it has already been published, please also adjust the filename, so the id matches the actual name, thanks!